notify/grafana_alertmanager.go

```diff
@@ -511,7 +497,7 @@ func (am *GrafanaAlertmanager) PutAlerts(postableAlerts amv2.PostableAlerts) err
 			am.Metrics.Resolved().Inc()
 		}
 
-		if err := validateAlert(alert); err != nil {
+		if err := alert.Validate(); err != nil {
```

Comment:
This is covered with existing tests [here](https://github.com/grafana/alerting/blob/1d58deb6074c6985fdc21e0e514a87a6f9e1dd3e/notify/grafana_alertmanager_test.go#L39).